api/envoy/extensions/filters/http/lua/v3/lua.proto

```diff
@@ -58,6 +58,23 @@ message Lua {
   // The default Lua code that Envoy will execute. If no per route config is provided
   // for the request, this Lua code will be applied.
   config.core.v3.DataSource default_source_code = 3;
+
+  // Optional additional prefix to use when emitting statistics. This allows to distinguish
+  // emitted statistics between configured ``lua`` filters in an HTTP filter chain. For example:
```

Comment:
Specify what the default is which I think is 'lua' ? Or maybe the default is empty? Unclear.

Reply:
The default is `lua`, updated the doc to be more explicit